### PR TITLE
Add light dark themes

### PR DESCRIPTION
**Pull request by stepiro**
Added roboto font.
min api 16.

### CODE REVIEW

File: demo/README.md [hunk 1]
```diff
@@ -5,22 +5,28 @@ This example app demonstrates how to integrate Android Payment Highway SDK for a
...
+
 For more info see the code [here]().
 
 In client side you need to implement the BackendAdapter interface. 
 
 The demo includes an implementation example for a `BackendAdapter` in the module `backendadapterexample`.
 
+## Environment
+
+Payment Highway provide a [development `sandbox`](https://dev.paymenthighway.io/#development-sandbox) and a `production` environments.
```
Comment (Swie) [inline demo/README.md]: provides

File: demo/README.md [hunk 2]
```diff
@@ -5,22 +5,28 @@ This example app demonstrates how to integrate Android Payment Highway SDK for a
...
+
 ## Add a credit card
 
 In order to add credit card and get a payment token you need a `PaymentContext`.
 
-PaymentHighway Android SDK (debug) in connected with the [Payment Highway development sandbox](https://dev.paymenthighway.io/#development-sandbox).
-
-Therefore the demo app uses the provided sandbox merchant id and account id.
+The demo app run in the `Sandbox` environment therefore uses the provided [sandbox `merchant id` and `account id`](https://dev.paymenthighway.io/#development-sandbox).
```
Comment (Swie) [inline demo/README.md]: run**s** in the `Sandbox` environment **and** therefore

File: demo/README.md [hunk 3]
```diff
@@ -91,6 +97,36 @@ You can listen when the card data is valid implementing in your Activity (for ex
     }  
 ```
 
+There is a helper where you can listen passing a lambda:
```
Comment (Swie) [inline demo/README.md]: I think this sentence is missing some words, what do you mean?

listen XXX by passing a lambda ?

File: demo/README.md [hunk 4]
```diff
@@ -91,6 +97,36 @@ You can listen when the card data is valid implementing in your Activity (for ex
...
+    mCardInputWidget = findViewById<AddCardWidget>(R.id.add_card_widget)
+
+    mCardInputWidget.setValidationListener {
+        isValidDidChange {
+            mAddCardButton.isEnabled = it
+        }
+    }
+```
+
+You can provide images for card text inputs implementing in your Activity the EditTextImageProvider interface:
```
Comment (Swie) [inline demo/README.md:111]: card text inputs by implementing the EditTextImageProvider interface in your Activity:

File: demo/README.md [hunk 5]
```diff
@@ -120,31 +156,50 @@ SDK provide Styles for the TextInputLayout, for the EditText and for the ErrorTe
...
+    <color name="primaryActiveForegroundColor">#de000000</color>
+    <color name="phPrimaryForegroundColor">#9d00ba</color>
+    <color name="phPrimaryForegroundColorDark">#d261e7</color>
+    <color name="hintColor">#ababab</color>
+    <color name="errorForegroundColor">#ff0000</color>
 ```
 
 You can customize the add card field either overriding the colors in your app or overriding the style(s) and define your own appearence.
+
+In the demo there is a definition of a dark theme as a example. You can try it just uncommenting the definition of the colors in the file color.xml. You need set the variable `val themeDark = true` in the file AddCardActivity; this is needed to return different images for the card inputs.
```
Comment (Swie) [inline demo/README.md]: - an example
- try it just by uncommenting
- you need to set the variable

File: demo/src/main/java/io/paymenthighway/demo/AddCardActivity.kt [hunk 6]
```diff
@@ -81,11 +110,24 @@ class AddCardActivity : AppCompatActivity(), ValidationListener {
...
 
+    override fun imageDrawable(type: EditTextType): Int = getDrawableId(type)
+
+    private fun hideKeyboard() {
+        if (currentFocus == null) return
+        val imm = this.getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
+        imm.hideSoftInputFromWindow(currentFocus!!.windowToken, 0)
+    }
+
+    private fun getDrawableId(type: EditTextType): Int = when (type) {
```
Comment (Swie) [inline demo/src/main/java/io/paymenthighway/demo/AddCardActivity.kt]: Was this related to some limitations? Feels a bit weird having to care about theme colors in the code.
Or is it necessary for dynamic theme changes perhaps?

**Review by Swie — CHANGES_REQUESTED** [review]

File: demo/README.md (hunk 1 above)
Comment (stepiro) [reply demo/README.md]: fixed thanks

File: demo/README.md (hunk 2 above)
Comment (stepiro) [reply demo/README.md]: fixed, thanks

File: demo/README.md (hunk 4 above)
Comment (stepiro) [reply demo/README.md:111]: fixed

**Review by Swie — APPROVED** [review]

**Pull request merged**